lib/ExportsStorage.js

```diff
@@ -52,6 +52,17 @@ export default class ExportsStorage {
 
             this.db.run(
               `
+                CREATE VIRTUAL TABLE exports_fts USING fts4(
```

Comment:
sqlite's full text search requires a virtual table and using full text search performs better than `LIKE 'foo%'`. I'm not too familiar with sqlite, but it's possible we don't need a separate table at all, but I figured this way I don't break anything.